tas/templates/tas/profile_edit.html

```diff
@@ -55,5 +63,29 @@ <h2>Edit Profile <small>| {{ user.username }} </small></h2>
 {% endblock %}
 
 {% block scripts %}
-<script type="text/javascript" src="{% static 'tas/js/institution_departments.js' %}"></script>
+<script type="text/javascript">
+function profile_title_update(value) {
```

Comment:
As a general note, I think it's nice to separate Javascript and Python functions w/ different syntax. Any JS codebase you will see nowadays typically settles on the camelCase style for everything, while Python uses snake_case. So this would be profileTitleUpdate(value) -- same goes for variable names.